homeassistant/components/generic_thermostat/config_flow.py

```diff
@@ -21,13 +21,18 @@
     CONF_COLD_TOLERANCE,
     CONF_HEATER,
     CONF_HOT_TOLERANCE,
+    CONF_MAX_TEMP,
     CONF_MIN_DUR,
+    CONF_MIN_TEMP,
     CONF_PRESETS,
     CONF_SENSOR,
     DEFAULT_TOLERANCE,
     DOMAIN,
 )
 
+DEFAULT_MIN_TEMP = climate.const.DEFAULT_MIN_TEMP
```

Comment:
Linter wont let me import these directly, so I have to import climate then set them.

```
W7424: Import should be using the component root (hass-component-root-import)
```

Reply:
`from ...climate import DEFAULT_MIN_TEMP`. 

You need to import from `climate` platform root so don't import from its `const.py` file.

Reply:
@gjohansson-ST Thanks, I couldn't quite figure out what that lint rule was trying to get me to do.